src/controllers/user-controller.js

```diff
@@ -0,0 +1,27 @@
+var userService = require('../services/user-service');
+
+function getUsers(req, res, next) {
+    userService.getUsers()
+    .then(function(users) {
+        res.json(users);
+    })
+    .catch(next);
+}
+
+function getUserById(req, res, next) {
```

Comment:
'next' is defined but never used.<br>'res' is defined but never used.<br>'req' is defined but never used.